pyproject.toml

```diff
@@ -58,12 +57,6 @@ optional-dependencies = {dev = [
     "sphinx_autodoc_typehints",
     "sphinxcontrib-video",
     "types-pytz",
-], pyqt = [
-    "PyQt5 >= 5.12.3, != 5.15.0",
-], pyside = [
-    "PySide2 >= 5.13.2, != 5.15.0",
-], qt = [
-    "epitools[pyside]",
```

Comment:
This was something @p-j-smith introduced in `btrack` https://github.com/lowe-lab-ucl/napari-btrack/pull/59. I mimicked here, but didn't fully understand the justification if I'm honest.

Reply:
Yea, I think this can be removed, at least for now. Epitools can run with whatever Qt backend is installed by the user as far as I can tell.

Reply:
Okay sounds good